service/src/main/java/bio/terra/tanagra/app/controller/AnnotationsV2ApiController.java

```diff
@@ -69,6 +72,18 @@ public ResponseEntity<ApiAnnotationV2> createAnnotation(
         toApiObject(annotationService.getAnnotation(studyId, cohortId, newAnnotationId)));
   }
 
+  @Override
+  public ResponseEntity<ApiExportFile> exportAnnotationValues(String studyId, String cohortId) {
+    accessControlService.throwIfUnauthorized(
+        SpringAuthentication.getCurrentUser(), READ, COHORT, new ResourceId(cohortId));
```

Comment:
This permission seem weird because the cohort permission gives you access to all of the review data as a whole (and nothing else about cohorts?) but not to individual reviews. Maybe renaming to `ANNOTATION_EXPORT` or something like that would make it seem more sensible?

Reply:
Discussed offline. Left `READ` since it's cleaner to use `READ` rather than a separate `EXPORT` permission.

Changed to `ANNOTATION`. (Is that what you wanted?) In whole file, changed `ANNOTATION` to check cohort instead of annotation, since we don't plan on enforcing access control at annotation granularity.

Reply:
Yeah, this is exactly what I was thinking.